incubator/hnc/pkg/reconcilers/hnc_config_test.go

```diff
@@ -312,3 +332,18 @@ func removeHNCConfigType(ctx context.Context, apiVersion, kind string) {
 		return updateHNCConfig(ctx, c)
 	}).Should(Succeed())
 }
+
+func createCronTabCRD(ctx context.Context) {
+	crontab := v1beta1.CustomResourceDefinition{}
+	crontab.Kind = "CronTab"
+	crontab.APIVersion = "stable.example.com/v1"
+	crontab.Name = "crontabs.stable.example.com"
+	crontab.Spec.Group = "stable.example.com"
+	crontab.Spec.Versions = append(crontab.Spec.Versions, v1beta1.CustomResourceDefinitionVersion{Name: "v1", Served: true, Storage: true})
+	crontab.Spec.Names.Kind = "CronTab"
+	crontab.Spec.Names.Plural = "crontabs"
+	crontab.Spec.Names.Singular = "crontab"
```

Comment:
Consider using the Go struct initializer, eg see line 262 in this file.

Reply:
Thanks for the suggestion! Done.